debug/testlib.py

```diff
@@ -276,11 +277,11 @@ def __init__(self, address):
 class Gdb(object):
     logfile = tempfile.NamedTemporaryFile(prefix="gdb", suffix=".log")
     logname = logfile.name
+    print "GDB Temporary file: %s" % logname
```

Comment:
I'd call these
"OpenOCD logfile" and "GDB logfile". At least be consistent in capitalization and temp/temporary, logfile/file